### PR TITLE
[Android/Test] update testcases @open sesame 9/27 15:35

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. update testcases for each java class
2. add flag to generate coverage report

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: api/android/api/build.gradle [hunk 1]
```diff
@@ -45,6 +45,9 @@ android {
         }
     }
     buildTypes {
+        debug {
```
Comment (leemgs) [inline api/android/api/build.gradle]: Looks. good. 

@jaeyun-jung  BTW, does the existing "**TAOS/pr-audit-nnstreamer-apptest**" module check the change of the *.java code of this PR?

Comment (jaeyun-jung) [reply api/android/api/build.gradle]: @leemgs  CI just checks build failure.

Comment (leemgs) [reply api/android/api/build.gradle]: @jaeyun-jung  Synched.

Comment (leemgs) [reply api/android/api/build.gradle]: @jaeyun-jung , Can we reproduce the build and run test with the existing guide (e.g., README.md)?
* https://github.com/nnsuite/nnstreamer/blob/master/api/android/README.md

Comment (jaeyun-jung) [reply api/android/api/build.gradle]: Yes, we can. The test result and library file will be copied into ```android_lib``` directory.

**Pull request merged**